.travis.yml

```diff
@@ -1,7 +1,14 @@
-branches:
-  only:
-  - master
-install: true
-script: ".travis/build.sh"
-cache:
-  dates.txt
+jobs:
```

Comment:
I've never used `jobs` before... I'm inclined to say it's overkill here? Could we instead use just `os` [like the imagej-launcher project does](https://github.com/imagej/imagej-launcher/blob/dead69b6780f76803da2b4fa5500a129c08940c1/.travis.yml)? And then the shell script just has a small amount of case logic for things like installing `dmgbuild` only if the platform is macOS? I don't feel super strongly, though, and I'm not trying to create more work if you substantially prefer this `jobs` approach.